doc/docs/Release_Notes.md

```diff
@@ -2,30 +2,31 @@
 # Release Notes
 ---
 
-Here, we describe what has changed between releases of the [Meep](index.md) package. You can also refer to the `NEWS` file in the Meep package or the `ChangeLog` file for a more detailed listing.
```

Comment:
My inclination would be to delete `Release_Notes.md` and just link to `NEWS.md` on github.